aspnetcore/fundamentals/environments.md

```diff
@@ -82,7 +82,7 @@ Changes made to project profiles may not take effect until the web server is res
 
 ### Production
 
-The production environment should be configured to maximize security, performance, and application robustness. Some common settings that a production environment might have that would differ from development include:
+The production environment should be configured to maximize security, performance, and application robustness. Some common settings that a production environment might have, that would differ from development, include:
```

Comment:
The word "that" usually introduces essential information, so it wouldn't normally be set off by commas. Could go with "which" here and commas. Another option is just to eliminate that phrase, since it's clearly describing a production environment.

> Some common settings that a production environment might have include: